### DISCUSSION TITLE
After installing the module on a clean install on localhost, it gets this error.

### DISCUSSION THREAD

**Discussion by compsoul** [PrestaShop/PrestaShop#34306] [Q&A]
![obraz](https://github.com/PrestaShop/PrestaShop/assets/21887883/cd3d133b-ae12-47c4-8ae1-8bf29e622ea9)

**Comment by kpodemski** [comment]
Hello @compsoul 

Did you run `composer install` after cloning the repository to your `/modules` directory?

**Reply by compsoul** [reply]
Not for the module folder, but for the folder "demosymfonyform" according to the instructions.

**Reply by kpodemski** [reply]
oh, I see what could go wrong, I think you've tried it on PrestaShop 8.1 but the GeoCordinatesType has been added to the PrestaShop 9 (develop branch) 🤔

**Reply by compsoul** [reply]
It seems that something is on your side, I have a similar problem with another module. But I'm still checking if it's not my fault sometimes.

**Reply by compsoul** [reply]
And so I installed on the latest version of PS and localhost

**Reply by kpodemski** [reply]
on the develop branch, there's a form type `GeoCoordinatesType` available:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Form/Admin/Type/GeoCoordinatesType.php

**Reply by compsoul** [reply]
I know but I can't use it because I'm writing the module under PS8.0 so I won't add in the instructions for someone to manually add the file from GIT. Unless the overwrite mechanism works.

**Reply by kpodemski** [reply]
if you create a module for PS8, simply edit the demo module and not use `GeoCoordinatesType` field

eventually, you have to adjust the module to your needs

**Reply by compsoul** [reply]
Only that I in the module needs this type. Thanks for your help, I consider the submission closed and you would need to remove this from the example or inform that it only works on the dev version.

**Reply by kpodemski** [reply]
You can always put this type into your module, no need for override, you can use your own types declarations :)

**Reply by compsoul** [reply]
Yes I know, however I guess it makes no sense to make an unnecessary dependency when as I understand in version 9 it will be a native functionality?
